types/keypair/private_key.go

```diff
@@ -10,6 +11,7 @@ import (
 type PrivateKeyInternal interface {
 	PublicKeyBytes() []byte
 	Sign(mes []byte) ([]byte, error)
+	ToPem() ([]byte, error)
```

Comment:
Propose to call method `PemEncode`

Reply:
It should involve additional refactoring to rename appropriate methods:
keypair.NewPrivateKeyFromFile
ed25519.NewPrivateKeyFromFile
ed25519.NewPrivateKeyFromPEM
secp256k1.NewPrivateKeyFromPemFile
secp256k1.NewPrivateKeyFromPem

Reply:
is it some kind of related to the renaming of `ToPem()` method ?

Reply:
in my opinion, it's clear when naming is symmetry:
atm: <- FromPem;  -> ToPem
your proposal: <- PemDecode; -> PemEncode right?